newdle/client/package.json

```diff
@@ -28,7 +29,11 @@
     "semantic-ui-react": "^1.2.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.11.6",
     "@jedmao/redux-mock-store": "^3.0.5",
+    "@lingui/cli": "^2.9.2",
+    "@lingui/macro": "^2.9.2",
```

Comment:
What do you think about using lingui 3?

https://lingui.js.org/releases/migration-3.html

Reply:
3.0 docs are here: https://js-lingui-git-next.lingui-js.now.sh/

Reply:
When I was through the installation setup, I didn't add any version so it installed the latest one. Maybe the lingui 3 is still in beta?
I had a look into the versions available and it's not the latest one.
https://www.npmjs.com/package/@lingui/cli
https://www.npmjs.com/package/@lingui/macro

Reply:
Yeah, looks like it's only in `@next`... not super clear from their website nor issue tracker...

Let's see if they recommend using 2.x or 3.x in new projects: https://github.com/lingui/js-lingui/issues/334#issuecomment-697811138

Reply:
some guy in the issue tracker:

> I've asked the same question and was recommended to use 3.0.0-13

Reply:
Ok, I will try with lingui 3 then :)

Reply:
due to lingui/js-lingui#707 (mentioned in a lter comment) it may be safer  to stick with 2 for now :) also since hacktoberfest starts soon: if we can get this PR ready and merged soon, we can open individual issues to add i18n to the various other areas, and benefit from people looking for small hacktoberfest issues that don't require much effort (ie no UI or new feature development)

Reply:
ok, perfect! I have planned to push it today and make the needed changes before Hacktoberfest :)